src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -338,6 +338,14 @@
     short-summary: The ARM Id or name of the subnet for the vnet. If omitted 'default' will be used
   - name: --protocol-types
     short-summary: Space seperated list of protocols that the volume can use
+  - name: --volume-type
+    short-summary: Whether the volume should be a data protection volume ("DataProtection")
+  - name: --endpoint-type
+    short-summary: Whether the volume is source ("src") or destination ("dst")
```

Comment:
There are two questions here:
1. Can you use get_enum_type() for allowed values?
2. With this parameter, I can create a source  or destination volume, how can I associate the source and destination?

Reply:
I suggest we do this change for next version/update that will come very soon.